conf/app.routes

```diff
@@ -87,9 +87,13 @@ POST        /:taxYear/annual-allowance/pension-provider-paid-tax-amount
 GET         /:taxYear/annual-allowance/pension-scheme-tax-reference                         controllers.pensions.annualAllowance.PensionSchemeTaxReferenceController.show(taxYear: Int, pensionSchemeTaxReferenceIndex: Option[Int])
 POST        /:taxYear/annual-allowance/pension-scheme-tax-reference                         controllers.pensions.annualAllowance.PensionSchemeTaxReferenceController.submit(taxYear: Int, pensionSchemeTaxReferenceIndex: Option[Int])
 
+
+###########################     Income from Pension pages     ####################################
```

Comment:
Minor thing, but could you move this bit below the pstr route as that belongs with the annual allowance stuff

Reply:
done